From 32bdfc0f1800c14baee5ebe0880d047148b8cea2 Mon Sep 17 00:00:00 2001
From: Wes Haggard <Wes.Haggard@microsoft.com>
Date: Fri, 7 Sep 2018 18:15:25 -0700
Subject: [PATCH] Update to released SDK

As part of this we need to disable a roslyn analyzer to workaround
https://github.com/dotnet/roslyn-analyzers/issues/1804.
---
 CodeAnalysis.ruleset | 6 +++++-
 DotnetCLIVersion.txt | 2 +-
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/CodeAnalysis.ruleset b/CodeAnalysis.ruleset
index 2bf447f3f6..8920fd2575 100644
--- a/CodeAnalysis.ruleset
+++ b/CodeAnalysis.ruleset
@@ -35,4 +35,8 @@
     <Rule Id="CA5350" Action="Error" /> <!-- Do not use Weak/Broken cryptographic algorithms -->
     <Rule Id="CA5351" Action="Error" />
   </Rules>
-</RuleSet>
+  <Rules AnalyzerId="Microsoft.AnalyzerPowerPack.CSharp" RuleNamespace="Microsoft.AnalyzerPowerPack.CSharp">
+    <!-- Disable analyzer warning 1821 until https://github.com/dotnet/roslyn-analyzers/issues/1804 is resolved -->
+    <Rule Id="CA1821" Action="None" />
+  </Rules>
+</RuleSet>
\ No newline at end of file
diff --git a/DotnetCLIVersion.txt b/DotnetCLIVersion.txt
index 3aa4b00373..6f1c03c6a3 100644
--- a/DotnetCLIVersion.txt
+++ b/DotnetCLIVersion.txt
@@ -1 +1 @@
-2.1.401-preview-009081
+2.1.401
\ No newline at end of file
-- 
2.18.0

